repos/system_upgrade/el9toel10/actors/checkoldxfs/libraries/checkoldxfs.py

```diff
@@ -0,0 +1,141 @@
+from leapp import reporting
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api
+from leapp.models import XFSInfoFacts
+
+RHEL_9_TO_10_BACKUP_RESTORE_LINK = 'https://red.ht/rhel_9_to_10_backup_restore_xfs'
```

Comment:
ah :D we forgot to drop it. nevermind. merging anyway. can be cleaned in furure.